docs/coronagraphs.rst

```diff
@@ -108,13 +108,13 @@ Again we will compare the execution time with the FFT case.::
         # Re-cast as MFT coronagraph with annular diaphragm FPM
         matrixFTcoron_annFPM_osys = poppy.MatrixFTCoronagraph( fftcoron_annFPM_osys, occulter_box=diaphragm.uninverted_optic.radius_inner )
         t0_fft = time.time()
-        annFPM_fft_psf, annFPM_fft_interm = fftcoron_annFPM_osys.calc_psf(wavelen, display_intermediates=True,\
+        annFPM_fft_psf, annFPM_fft_interim = fftcoron_annFPM_osys.calc_psf(wavelen, display_intermediates=True,\
                                                                  return_intermediates=True)
         t1_fft = time.time()
         
         plt.figure()
         t0_mft = time.time()
-        annFPM_mft_psf, annFPM_mft_interm = matrixFTcoron_annFPM_osys.calc_psf(wavelen, display_intermediates=True,\
+        annFPM_mft_psf, annFPM_mft_interim = matrixFTcoron_annFPM_osys.calc_psf(wavelen, display_intermediates=True,\
                                                                      return_intermediates=True)
```

Comment:
As the whitespace for the second line didn't line up consistently for these two calls, I didn't adjust it for this change.